io.gitlab.android_translation_layer.BaseApp.yml

```diff
@@ -0,0 +1,261 @@
+  - '/lib/libopenxr_loader.so*'
+  - '/lib/libportal-*'
+  - /lib/cmake
+  - /lib/girepository-1.0
+  - /lib/java/core-all_classes.jar
+  - /lib/java/dx.jar
+  - /lib/pkgconfig
+  - /lib/python3.11
+  - /share/doc
+  - /share/gir-1.0
+  - /share/man
+  - /share/vala
+modules:
+  - name: gn
+    buildsystem: simple
+    build-options:
+      env:
+        CXX: c++
+    build-commands:
+      - echo "#define LAST_COMMIT_POSITION_NUM 0" > src/gn/last_commit_position.h
+      - echo "#define LAST_COMMIT_POSITION \"\"" >> src/gn/last_commit_position.h
+      - python3 ./build/gen.py --no-last-commit-position --no-static-libstdc++ --no-strip --allow-warnings
+      - ninja -C out
+      - install -Dm755 out/gn /app/bin/gn
+    sources:
+      - type: git
+        url: https://gn.googlesource.com/gn
+        commit: e4702d7409069c4f12d45ea7b7f0890717ca3f4b
+  - name: python3-typogrify
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "typogrify" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/da/ed/1da76d11aa858ee23dac5b52d9ac2db7df02b89f7679d5d8970bcd44b59c/smartypants-2.0.1-py2.py3-none-any.whl
+        sha256: 8db97f7cbdf08d15b158a86037cd9e116b4cf37703d24e0419a0d64ca5808f0d
+      - type: file
+        url: https://files.pythonhosted.org/packages/8a/bf/64959d6187d42472acb846bcf462347c9124952c05bd57e5769d5f28f9a6/typogrify-2.0.7.tar.gz
+        sha256: 8be4668cda434163ce229d87ca273a11922cb1614cb359970b7dc96eed13cb38
+  - name: libmd
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://archive.hadrons.org/software/libmd/libmd-1.1.0.tar.xz
+        sha256: 1bd6aa42275313af3141c7cf2e5b964e8b1fd488025caf2f971f43b00776b332
+  - name: libbsd
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://gitlab.freedesktop.org/libbsd/libbsd.git
+        tag: '0.12.2'
+  - name: libportal
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/flatpak/libportal.git
+        tag: '0.7.1'
+  - name: openxr
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/KhronosGroup/OpenXR-SDK.git
+        tag: 'release-1.1.36'
+  - name: skia
+    buildsystem: simple
+    build-commands:
+      - gn gen 'out/linux/x64' --args='is_official_build=true skia_enable_tools=false target_os="linux" target_cpu="x64" skia_use_icu=false skia_use_sfntly=false skia_use_piex=true skia_use_system_harfbuzz=true skia_use_system_expat=true skia_use_system_freetype2=true skia_use_system_libjpeg_turbo=true skia_use_system_libpng=true skia_use_system_libwebp=true skia_use_system_zlib=true skia_enable_gpu=true extra_cflags=[ "-DSKIA_C_DLL" ] linux_soname_version="99.9"'
+      - ninja -C 'out/linux/x64' SkiaSharp
+      - install -D out/linux/x64/libSkiaSharp.so.99.9 /app/lib/libSkiaSharp.so
```

Comment:
so what happen on aarch64?

Reply:
I have fixed it now by getting the correct value from `$FLATPAK_ARCH`